userland/examples/rf233/rf233.c

```diff
@@ -377,12 +379,11 @@ int rf233_setup(void) {
   volatile uint8_t radio_state;  /* don't optimize this away, it's important */
 
   /* init SPI and GPIOs, wake up from sleep/power up. */
-
   spi_init();
   // RF233 expects line low for CS, this is default SAM4L behavior
   //spi_set_chip_select(3);
   // POL = 0 means idle is low
-  spi_set_chip_select(3);
+  spi_set_chip_select(3); 
   spi_set_polarity(0);
   // PHASE = 0 means sample leading edge
   spi_set_phase(0);
```

Comment:
seems that the app uses 400000 as it's rate, as it's set a little below that line. I've removed the excess whitespace in the file.